docker/cortx-deploy/build.sh

```diff
@@ -97,6 +94,13 @@ echo cortx-rgw:"$(awk -F['-'] '/'ceph-base'/{ print $5  }' RELEASE.INFO | cut -d
 echo cortx-re:"$(git rev-parse --short HEAD)",
 }
 
+function setup_local_image_registry {
+docker rm -f local-registry
```

Comment:
can we handle failure of commands in this function with error message? and please align code in function properly

Reply:
Fixed.